src/client/src/app/modules/offline/services/content-manager/content-manager.service.ts

```diff
@@ -81,9 +84,31 @@ export class ContentManagerService {
         this.toasterService.info(this.resourceService.messages.smsg.m0053);
         this.downloadEvent.emit('Download started');
       }),
-      catchError((err) => {
+      catchError((err: any) => {
         if (err.error.params.err === 'LOW_DISK_SPACE') {
-          this.downloadFailEvent.emit(this.failedContentName);
+          const popupInfo: any = {
+            failedContentName: this.failedContentName,
+          };
+
+          this.systemInfoService.getSystemInfo().subscribe((info: any) => {
+
```

Comment:
Add another api to get the suggested drive since it should cover the following conditions

1. We should not show used drives / filled drives 
2. If all drives are full it should return empty that means we should show that the system has no space
3. and should return all the available drives so that user can select wherever he want to store